builtin/v11/miner/miner_state.go

```diff
@@ -170,8 +170,8 @@ type SectorOnChainInfo struct {
 	InitialPledge         abi.TokenAmount // Pledge collected to commit this sector
 	ExpectedDayReward     abi.TokenAmount // Expected one day projection of reward for sector computed at activation time
 	ExpectedStoragePledge abi.TokenAmount // Expected twenty day projection of reward for sector computed at activation time
-	ReplacedSectorAge     abi.ChainEpoch  // Age of sector this sector replaced or zero
-	ReplacedDayReward     abi.TokenAmount // Day reward of sector this sector replace or zero
+	PowerBaseEpoch        abi.ChainEpoch  // Age of sector this sector replaced or zero
+	ReplacedDayReward     abi.TokenAmount // Day reward of sector this sector replaced or zero
```

Comment:
Update the comments too.
Rename ReplacedDayReward if it's subsequently renamed in https://github.com/filecoin-project/builtin-actors/pull/1229.